suite-native/device/src/components/UninitializedDeviceModalAppendix.tsx

```diff
@@ -0,0 +1,86 @@
+import { RequireExactlyOne } from 'type-fest';
+
+import { VStack, HStack, Text, Box } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Icon, IconName } from '@suite-common/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+import { TxKeyPath } from '@suite-native/intl';
+
+const listItemStyle = prepareNativeStyle(() => ({
+    width: '90%',
+}));
+
+const iconBackgroundStyle = prepareNativeStyle<{ color: Color }>((utils, { color }) => ({
+    backgroundColor: utils.colors[color],
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: utils.spacings.extraLarge,
+    height: utils.spacings.extraLarge,
+    borderWidth: color ? utils.borders.widths.small : 0,
+    borderColor: utils.colors.borderElevation1,
+    borderRadius: 12,
+}));
+
+type ListItemProps = RequireExactlyOne<
+    {
+        iconBackgroundColor?: Color;
+        translationKey: TxKeyPath;
+        iconColor?: Color;
+        iconName: IconName;
+        iconNumber: number;
+    },
+    'iconName' | 'iconNumber'
+>;
+
+const ListItem = ({
+    iconName,
+    iconColor,
+    iconNumber,
+    iconBackgroundColor = 'backgroundTertiaryDefaultOnElevation1',
+    translationKey,
+}: ListItemProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <HStack spacing={12} alignItems="center">
+            <Box style={applyStyle(iconBackgroundStyle, { color: iconBackgroundColor })}>
+                {iconNumber && <Text color={iconColor}>{iconNumber}</Text>}
+                {!iconNumber && iconName && (
```

Comment:
these should be mutually exclusive based on RequireExactlyOne  props, shouldnt they?
```suggestion
                {iconName && (
```

Reply:
I tend to not rely on typescript in cases like this and validate whenever it makes sense, but here it makes not much sense and I wouldn't have written the condition if I hadn't written it before the type... 6dddbfb38f53c2ff0a882ba5f8079197ecb44cf9